source/PreferencesPanel.cpp

```diff
@@ -61,6 +61,9 @@ namespace {
 	const string SCROLL_SPEED = "Scroll speed";
 	const string FIGHTER_REPAIR = "Repair fighters in";
 	const string SHIP_OUTLINES = "Ship outlines in shops";
+
+	// How many pages of settings there are.
+	const int pageCount = 1;
```

Comment:
Maybe this asked a bit too much, but maybe this variable could be auto set?
Counting the settings and at a certain number a page break appears.
If you you want to do this even further, dont page break in a category but after a category.

Not something neccessary though. Your decision.

Reply:
Doing that would require traversing the list of settings, which would require moving them out of DrawSettings. Traversing them isn't really that expensive, since this is the preferences panel anyway, but I don't know if we want to move it elsewhere (probably the anonymous namespace.)

Reply:
Whilst conceptually it would be nice to have that auto calculated, we'll probably be going from 1 to 2 in the near future and staying at that forever pretty much so I dont think its worth it